app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
We don't include :confirmable module for users here so we should either change it here or delete columns for confirmable module from migration.

Reply:
done